### PR TITLE
Fix/tests default values for onpremise

### PR DESCRIPTION
**Pull request by AlvaroVega**
Align orchestrator tests to run in docker/onpremise platform

### CODE REVIEW

File: src/tests/api/test_api.py [hunk 1]
```diff
@@ -3328,9 +3331,10 @@ def test_reset_metrics_ok(self):
     test_SubServiceIoTADevice.test_post_ok2()
     test_SubServiceIoTADevice.test_post_ok3()
 
-    test_SubServiceIoTAService = Test_SubServiceIoTAService_RestView()
-    test_SubServiceIoTAService.test_post_ok()
-    test_SubServiceIoTAService.test_post_ok2()
+    # RegisterService was used by legacy BlackButton scenario
```
Comment (fgalan) [inline src/tests/api/test_api.py]: Leftover comments?

Comment (AlvaroVega) [reply src/tests/api/test_api.py]: Provision of blackbutton was handlec by ORC. Who knows...

Comment (fgalan) [reply src/tests/api/test_api.py]: Ok, good point to keep it.

However, I'd suggest to add a comment explaining why this is commented, e.g.:

```
# Blackbutton test has been disabled given we are no longer supporting this bla bla
```

**Comment by fgalan** [comment]
LGTM

**Pull request merged**